<launch>
  <param name="global_frame_id" value="/base_link" />	
<!--publish a point cloud to "cloud_pcd" topic-->
<!--output: cloud_pcd -->
<node pkg="point_cloud_mapping" type="pcd_to_msg_node" name="pcd_to_msg_node" output="screen" respawn="true"
      args="$(find cloud_tools)/data/box.pcd $(optenv period 5)" />

<!-- Box Filter, cuts out the box part of the pointcloud -->
<!--input: /multi_pcd_to_msg_node/cloud_pcd -->
<!--output: /shoulder_cloud_clipped -->
<node pkg="cloud_tools" type="box_filter_node" name="box_filter_node" output="screen">
  <remap from="~cloud_pcd" to="cloud_pcd" />
  <param name="output_cloud_topic" value="cloud_clipped" />
  <param name="box_min_x" value="0.0" />
  <param name="box_max_x" value="1.3" />
  <param name="box_min_y" value="-1.0" />
  <param name="box_max_y" value="1.0" />
  <param name="box_min_z" value="-0.5" />
  <param name="box_max_z" value="0.5" />
</node>
</launch>
